pages/common/xmllint.md

```diff
@@ -0,0 +1,27 @@
+# xmllint
+
+> XML parser and linter.
+
+- Return all nodes named "foo":
+
+`xmllint --xpath "//{{foo}}" {{source_file.xml}}`
+
+- Return as string the contents of first node named "foo":
+
+`xmllint --xpath "string//{{foo}}" {{source_file.xml}}`
+
+- Use other xpath (a syntax for navigating xml trees) expressions for more options in navigating xml tree:
+
+`xmllint --xpath "{{xpath_expression}}" {{source_file.xml}}`
+
+- Return human-readable (indented) xml from file:
+
+`xmllint --format {{source_file.xml}}`
+
+- Check that XML meets requirements of its built-in doctype. This is the part starting with <!DOCTYPE...>:
```

Comment:
@agnivade In which case, I think that `<!DOCTYPE...>` should be surrounded in code blocks.